io.github.amourspirit.LibrePythonista_PyEditor.metainfo.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="console-application">
+  <id>io.github.amourspirit.LibrePythonista_PyEditor</id>
+  <url type="homepage">https://github.com/Amourspirit/LibrePythonista_PyEditor</url>
+  <url type="bugtracker">https://github.com/Amourspirit/LibrePythonista_PyEditor/issues</url>
+  <update_contact>https://github.com/Amourspirit/LibrePythonista_PyEditor/issues/new</update_contact>
+  <name>LibrePythonista Py</name>
+  <summary>LibrePythonista ext python editor</summary>
+  <branding>
+    <color type="primary" scheme_preference="light">#dbf8ff</color>
+    <color type="primary" scheme_preference="dark">#194894</color>
+  </branding>
+  <metadata_license>MIT</metadata_license>
+  <project_license>MIT</project_license>
+  
+  <description>
+    <p>
+      This app is Python editor for LibreOffice using the LibrePythonista extension.
+    </p>
+  </description>
+  <icon type="stock">io.github.amourspirit.LibrePythonista_PyEditor</icon>
+  
+  <categories>
+    <category>Utility</category>
+    <category>TextTools</category>
+  </categories>
+  
+  <provides>
+    <binary>cell_edit</binary>
+  </provides>
+  <developer id="io.github.amourspirit">
+    <name>:Barry-Thomas-Paul: Moss</name>
+    <email>4193389+Amourspirit@users.noreply.github.com</email>
+  </developer>
+  <releases>
+    <release date="2024-11-24" version="v0.1.0">
+      <description>
+        <p>
+          Release 0.1.0: Initial Release
+        </p>
+      </description>
+    </release>
+  </releases>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://github.com/user-attachments/assets/bfe54760-84ee-48c7-b07a-b18431f81cdd</image>
```

Comment:
Where did you upload this screenshot and the one bellow it? Perhaps commit them to you git repo instead, and reference them here by their commit hash?

Reply:
this image link won't work, it redirects to a tokenised github link.